src/auth/auth.ts

```diff
@@ -179,6 +179,39 @@ class Auth implements FirebaseServiceInterface {
   };
 
   /**
+   * Exports the users (single batch only) with a size of maxResults and starting from
+   * the offset as specified by pageToken.
+   *
+   * @param {number=} maxResults The page size, 1000 if undefined. This is also the maximum
+   *     allowed limit.
+   * @param {string=} pageToken The next page token. If not specified, returns users starting
+   *     without any offset.
+   * @return {Promise<{users: UserRecord[], pageToken?: string}>} A promise that resolves with
+   *     the current batch of downloaded users and the next page token.
+   */
+  public listUsers(maxResults?: number, pageToken?: string): Promise<Object> {
```

Comment:
Should we define the type `ListUsersResult` for the return value? I noticed that it is mentioned in the d.ts files.

Reply:
Done.